### ISSUE TITLE
Image resize dimensions in the Dataloader

### ISSUE THREAD

**Issue by ErezYosef** [yhenon/pytorch-retinanet#211]
I wonder about the operation In the following lines:

https://github.com/yhenon/pytorch-retinanet/blob/ef2c16f4ad66b652b836be36465ee63c978f3194/retinanet/dataloader.py#L363-L364

I assume that it suppose to round up the image dimensions to the closest number divided by `32`.

But, when the dimensions **already** divided by `32` there is an *Extra* padding.
* For example for image in size 608x608px  :   `32 - rows%32` = `32-0` = `32` .
    So the image will pad with *Extra* `pad_w=32` pixels.

Suggested change for example:
```
 pad_w = ( 32 - rows%32 ) % 32
 pad_h = ( 32 - cols%32 ) % 32
```
 I would like to hear your opinion.
Erez.

**Comment by JuZiSYJ** [comment]
Hello，i also notice that
What is more, i wonder how can it feed the B*C*H*W data in to model, since the picture is not padding to the same size.

**Comment by junhai0428** [comment]
> pytorch-retinanet/retinanet/dataloader.py

I think the pictures have same size after collater operation in dataloader.
